egs/wsj/s5/steps/libs/common.py

```diff
@@ -65,6 +68,20 @@ def __call__(self, parser, namespace, values, option_string=None):
         else:
             setattr(namespace, self.dest, values)
 
+import contextlib
+@contextlib.contextmanager
```

Comment:
I'd prefer you make this a class with `__enter__` and `__exit__` methods, if there is an easy way to do that, and remove the contextlib dependency.    I.e. if the only purpose of contextlib was to save a few lines of code, then add a few lines and remove the dependency.  Otherwise at least explain what is going on.